docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,158 @@
+
+4. At the bottom of the task pane, choose the **Get My User Profile Information** button to initiate the SSO process.
+
+5. If a dialog window appears to request permissions on behalf of the add-in, this means that SSO is not supported for your scenario and the add-in has instead fallen back to an alternate method of user authentication. This may occur when the tenant administrator hasn't granted consent for the add-in to access Microsoft Graph, or when the user isn't signed into Office with a valid Microsoft account or Microsoft 365 Education or Work account. Choose the **Accept** button in the dialog window to continue.
+
+    ![Screenshot showing permissions requested dialog with Accept button highlighted.](../images/sso-permissions-request.png)
+
+    > [!NOTE]
+    > After a user accepts this permissions request, they won't be prompted again in the future.
+
+6. The add-in retrieves profile information for the signed-in user and writes it to the document. The following image shows an example of profile information written to an Excel worksheet.
+
+    ![Screenshot showing user profile information in Excel worksheet.](../images/sso-user-profile-info-excel.png)
+
+### Outlook
+
+Complete the following steps to try out an Outlook add-in.
+
+1. When the SSO configuration process completes, run the following command to build the project and start the local web server.
+
+    [!INCLUDE [alert use https](../includes/alert-use-https.md)]
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+2. Follow the instructions in [Sideload Outlook add-ins for testing](../outlook/sideload-outlook-add-ins-for-testing.md) to sideload the add-in in Outlook. Make sure that you're signed in to Outlook with a user that's a member of the same Microsoft 365 organization as the Microsoft 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso). Doing so establishes the appropriate conditions for SSO to succeed.
+
+3. In Outlook, compose a new message.
+
+4. In the message compose window, choose the **Show Taskpane** button in the ribbon to open the add-in task pane.
+
+    ![Screenshot showing highlighted add-in ribbon button in Outlook compose message window.](../images/outlook-sso-ribbon-button.png)
+
+5. At the bottom of the task pane, choose the **Get My User Profile Information** button to initiate the SSO process.
+
+6. If a dialog window appears to request permissions on behalf of the add-in, this means that SSO is not supported for your scenario and the add-in has instead fallen back to an alternate method of user authentication. This may occur when the tenant administrator hasn't granted consent for the add-in to access Microsoft Graph, or when the user isn't signed into Office with a valid Microsoft account or Microsoft 365 Education or Work account. Choose the **Accept** button in the dialog window to continue.
```

Comment:
```suggestion
6. If a dialog window appears to request permissions on behalf of the add-in, this means that SSO is not supported for your scenario and the add-in has instead fallen back to an alternate method of user authentication. This may occur when the tenant administrator hasn't granted consent for the add-in to access Microsoft Graph, or when the user isn't signed in to Office with a valid Microsoft account or Microsoft 365 Education or Work account. Choose **Accept** to continue.
```